packages/esm-patient-registration-app/src/patient-registration/input/custom-input/autosuggest/autosuggest.component.tsx

```diff
@@ -150,10 +161,11 @@ export const Autosuggest: React.FC<AutosuggestProps> = ({
   return (
     <div className={styles.autocomplete} ref={wrapper}>
       <label className="cds--label">{labelText}</label>
-      <Layer>
+      <Layer className={invalid ? styles.invalid : {}}>
```

Comment:
Please use the `classnames` library for this.

Reply:
Done